src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment: { name, email, body } }) => (
+  <>
```

Comment:
Here we have only one element. Therefore, we do not need an extra wrapper `<>...</>` of the react fragment.
Fix it everywhere.

```suggestion
```